app/src/main/res/navigation/nav_graph.xml

```diff
@@ -300,4 +309,31 @@
         android:id="@+id/exceptionsFragment"
         android:name="org.mozilla.fenix.exceptions.ExceptionsFragment"
         android:label="ExceptionsFragment" />
+    <dialog
+        android:id="@+id/createCollectionFragment"
+        android:name="org.mozilla.fenix.collections.CreateCollectionFragment"
+        android:label="fragment_create_collection"
+        tools:layout="@layout/fragment_create_collection" />
+    <dialog
+        android:id="@+id/quickSettingsSheetDialogFragment"
+        android:name="org.mozilla.fenix.settings.quicksettings.QuickSettingsSheetDialogFragment"
+        android:label="QuickSettingsSheetDialogFragment" >
+        <argument
+            android:name="url"
+            app:argType="string" />
+        <argument
+            android:name="isSecured"
+            app:argType="boolean" />
+        <argument
+            android:name="isTrackingProtectionOn"
+            app:argType="boolean" />
+        <argument
+            android:name="sitePermissions"
+            app:argType="mozilla.components.feature.sitepermissions.SitePermissions"
+            app:nullable="true" />
+        <argument
+            android:name="gravity"
+            app:argType="integer"
+            android:defaultValue="80" />
```

Comment:
Just to confirm, @ekager is this a valid gravity value?

Reply:
Yeah it's the Gravity.BOTTOM constant :)

Reply:
Awesome :)